ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/CommandParser.java

```diff
@@ -485,4 +528,49 @@ public String getProperty() {
       return value;
     }
   }
+
+  /**
+   * Represents ksqlDB DEFINE commands.
+   */
+  public static class SqlDefineVariableCommand extends SqlCommand {
+    private final String variable;
+    private final Object value;
+
+    SqlDefineVariableCommand(
+        final String command,
+        final String variable,
+        final Object value
```

Comment:
This is always passed in as a string, right? If so we should simplify this by making that explicit.